Sources/SwiftGenKit/Parsers/Colors/ColorsParser.swift

```diff
@@ -34,23 +34,33 @@ public enum Colors {
     var palettes = [Palette]()
     public var warningHandler: Parser.MessageHandler?
 
+    private static let subParsers: [ColorsFileTypeParser.Type] = [
+      CLRFileParser.self,
+      JSONFileParser.self,
+      TextFileParser.self,
```

Comment:
Now that I think about it, the `.txt` format for colors looks a lot like a YAML dictionary… so maybe in a future release we could deprecate the `.txt` extension and add a `YAMLFileParser` for colors instead, and tell users to just rename their `colors.txt` to `colors.yml` when doing the migration? Not for this PR obviously, but just a thought for the future of SwiftGen and for slowly deprecating a legacy historical proprietary format we created in the early days of SwiftGen 😛 )

Reply:
Sure. We can even add the `.txt` extension to the future yaml parser if we want. Or just drop it completely, no idea how many people ever used it?

Reply:
Yeah I think when times comes we should drop it completely. It was probably used by many in the early days of SwiftGen, so some legacy projects might still have kept using that format, but if all it takes is to change the extension of their file from `.txt` to `.yml`…

We'll have time to discuss that in the future when we decide to do a pass on things we want to deprecate or cleanup for future releases